src/PJ_misrsom.c

```diff
@@ -78,8 +78,11 @@ static XY e_forward (LP lp, PJ *P) {          /* Ellipsoidal, forward */
             sav = lampp;
             lamtp = lp.lam + Q->p22 * lampp;
             cl = cos(lamtp);
+#ifdef fixme
```

Comment:
Same as above. Has `lamtp` been used previously or is this just something that has been left in by mistake when originally implementing this projection?

Reply:
Similarly as PJ_lsat.c, the issues dates back to the initial commit ( https://github.com/OSGeo/proj.4/commit/991ed3603ecd46284b91bdbaaa39a01cfa6c0c70#diff-0dca5024bc8819892dd0d81ca8be2b90 ) , but this a "recent" file, probably a copy&paste from PJ_lsat

Reply:
Lines removed